R/deploy-site.R

```diff
@@ -0,0 +1,84 @@
+#' Deploy a pkgbown site on Travis-CI using Github Pages
+#'
+#' @inheritParams build_site
+#' @param tarball The location of the built package tarball. The default Travis
+#'   configuration for R packages sets `PKG_TARBALL` to this path.
+#' @param ssh_id The private id to use, a base64 encoded value, by default this
+#'   is read from the id_rsa environment variable. It is recommended you add this
+#'   key as a private environment variable in the Travis repository settings.
+#' @param ssh_id_file The file location to write the ssh private key obtained from `ssh_id`.
+#' @param repo_slug The `user/repo` slug for the repository.
+#' @param commit_message The commit message to be used for the commit.
+#' @param verbose Print verbose output
+#' @param ... Additional arguments passed to [build_site()].
+#' @export
+deploy_site <- function(
```

Comment:
Not sure how to effectively test this. Can we create a dummy repo and the test its existence?